ouroboros-network/src/Ouroboros/Network/Counter.hs

```diff
@@ -0,0 +1,8 @@
+module Ouroboros.Network.Counter (
+    Counter
+  ) where
+
+import Control.Tracer
+
+-- | Type alias to 
+type Counter m a = Tracer m a
```

Comment:
@newhoggy, does this still belong in this PR?

Reply:
I assumed it was an accident in my previous comment. (Serge, this PR is already Merged.)

https://github.com/input-output-hk/ouroboros-network/pull/2874#issuecomment-768667380

Reply:
Will be removed in a new PR.  Thanks!